### ISSUE TITLE
button has incorrect aria and is not keyboard accessible

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1351]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
- **The WAI-ARIA role(s) and/or attribute(s) button are not valid for the element label. See rule help for guidance.**

<img width="692" alt="screen shot 2018-10-26 at 3 49 24 pm" src="https://user-images.githubusercontent.com/40970507/47591901-dd5b2700-d936-11e8-9838-1029779164aa.png">

- **the add file button cannot be thrown using either enter or spacebar when testing in Chrome on macOS**

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset
